### PR TITLE
Add condition to check if checkbox is readonly

### PR DESCRIPTION
**Pull request by SpiderNeo1**
Add condition to check if the checkbox can be modified !

### CODE REVIEW

**Comment by SpiderNeo1** [comment]
It's ok ?

**Comment by SpiderNeo1** [comment]
Hello, 

When will you push my change ?

Thank you.

File: src/js/jquery.uniform.js [hunk 1]
```diff
@@ -127,25 +127,27 @@
...
-                isChecked = $el.is(":" + c);
-
-            if ($el.prop) {
-                // jQuery 1.6+
-                $el.prop(c, isChecked);
-            } else {
-                // jQuery 1.5 and below
-                if (isChecked) {
-                    $el.attr(c, c);
+        if(!$el.attr("readonly")) {
```
Comment (shehi) [inline src/js/jquery.uniform.js:130]: Even if checkbox is readonly, some disabled style should be applied to it. Can't leave a checkbox totally unmodified and destroy form's design unity.

Comment (SpiderNeo1) [reply src/js/jquery.uniform.js:130]: Hello,

No, of course you have to keep the design harmony, but you can not check / uncheck the box if it is in readonly

I will review the code.

**Pull request closed without merging**